TheJumpCloud/support
# UN-2212 Command Template for Removing Extra BitLocker Numerical Passwords

## Issues
* [UN-2212](https://jumpcloud.atlassian.net/browse/UN-2212) - Command Template for Removing Extra BitLocker Numerical Passwords

## What does this solve?
Adds a Powershell script to remove extra Bitlocker recovery passwords.

## Is there anything particularly tricky?
This PR should go after [UN-2288](https://jumpcloud.atlassian.net/browse/UN-2288).

## How should this be tested?
Bitlocker policy should be activated on a windows device.  After the OS drive is encrypted, in an Administrator Powershell command window type the following to create an extra Recovery Password:
`manage-bde -protectors -add -rp c:`
Then type the following to make sure there are two Numerical Passwords:
`manage-bde -protectors -get c:`
Run the command template from the Admin Portal to remove the extra password.  Run the previous command in a Powershell window to make sure there is only one Numerical Password.

## Screenshots


[UN-2212]: https://jumpcloud.atlassian.net/browse/UN-2212?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[UN-2288]: https://jumpcloud.atlassian.net/browse/UN-2288?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
